packages/clarity-js/src/layout/node.ts

```diff
@@ -104,6 +105,20 @@ export default function (node: Node, source: Source): Node {
                     break;
                 case "NOSCRIPT":
                 case "META":
+                    if (Constant.Property in attributes && Constant.Content in attributes) {
+                        var content = attributes[Constant.Content]
```

Comment:
We don't use var anymore. Use let for now. We will make a pass and clean everything else.